.env.template

```diff
@@ -1,7 +1,38 @@
+# AWS Secrets
+# These credentials are baked into the image to be used by the application
 export AWS_ACCESS_KEY_ID="tHiSiSyOuRAcCeSsKeY"
 export AWS_SECRET_ACCESS_KEY="tHiSiSyOuRAcCeSsSeCrEt"
-export DEV_AWS_ACCESS_KEY_ID="tHiSiSyOuRdEvAcCeSsKeY"
-export DEV_AWS_SECRET_ACCESS_KEY="tHiSiSyOuRdEvAcCeSsSeCrEt"
+
+# This is a non-standard name used in this repo;
+# it is tech debt and should be cleaned up to use the standard name
+export AWS_ACCESS_SECRET=$AWS_SECRET_ACCESS_KEY
+
+# These are only used by GitHub Actions, and use secrets of the same names
+# configured on GitHub; they are listed here only for the sake of completeness
+export DEV_AWS_ACCESS_KEY_ID=
+export DEV_AWS_SECRET_ACCESS_KEY=
+export STAGING_AWS_ACCESS_KEY_ID=
+export STAGING_AWS_SECRET_ACCESS_KEY=
+export PROD_AWS_ACCESS_KEY_ID=
+export PROD_AWS_SECRET_ACCESS_KEY=
+
+# These credentials are used by Packer to build the AMI
+# See the item "AWS Github/Terraform build access key" in Bitwarden
+export DEPLOY_AWS_ACCESS_KEY_ID="tHiSiSyOuRAcCeSsKeY"
+export DEPLOY_AWS_SECRET_ACCESS_KEY="tHiSiSyOuRAcCeSsSeCrEt"
+
+# AWS Region
+# For developers, it is easier to clean up if you use a different region than
+# we do for everything else; note that you'll need to switch regions in the
+# console to see your AMI
+# See also https://docs.aws.amazon.com/general/latest/gr/rande.html
 export AWS_REGION=us-east-1
+
+# Permanent environment name; must be one of [local, dev, staging, prod].
+# For developers, keep this as dev.
 export PERM_ENV=dev
+
+# Sentry DSN
+# Get this from Sentry. If the upload-service receives an empty string, it will
+# leave its Sentry integration unconfigured.
 export UPLOAD_SERVICE_SENTRY_DSN="https://EXAMPLE@EXAMPLE.ingest.sentry.io/EXAMPLE"
```

Comment:
I just realized this is incorrect; it is confusing the variable passed to packer (`SENTRY_DSN`) with the variable passed to `configure.sh` (`UPLOAD_SERVICE_SENTRY_DSN`).

Honestly, the way we're handling environment variables is so confusing. The sequence is at least:

GitHub secrets -> GitHub Actions -> packer ->  ansible -> `configure.sh`

In this case, while piping through the Sentry DSN to used by the upload service, the value is variously known as:

- `SENTRY_TOKEN` in GitHub (as an organization level secret!?)
- `SENTRY_DSN` during the GitHub Action: https://github.com/PermanentOrg/infrastructure/blob/82e3204545fe53dbbf7fb0b4469cc7a35ce209cc/.github/workflows/build-images.yml#L48
- `sentry_dsn` during the packer build: https://github.com/PermanentOrg/infrastructure/blob/82e3204545fe53dbbf7fb0b4469cc7a35ce209cc/images/image.json#L92
- `sentry_dsn` in the call to ansible: https://github.com/PermanentOrg/infrastructure/blob/82e3204545fe53dbbf7fb0b4469cc7a35ce209cc/images/image.json#L67
- `UPLOAD_SERVICE_SENTRY_DSN` in the ansible playbook: https://github.com/PermanentOrg/infrastructure/blob/main/provisioners/setup.yml#L91
- and, finally `SENTRY_DSN` by the service itself: https://github.com/PermanentOrg/infrastructure/blob/82e3204545fe53dbbf7fb0b4469cc7a35ce209cc/templates/etc/systemd/system/node-upload.service#L15

I'll fix this tomorrow.

Reply:
Fixed in commit c1e8232.